test/core/SuiteLocal.ts

```diff
@@ -51,12 +51,15 @@ export class SuiteLocal {
     // set the debug=1 in the environment to have the debug messages dumped during testing
     if (process.env['DEBUG'] || process.env['debug']) {
       this.session.channels.on('debug', (text, context, msec) => {
-        console.log(`[${msec}msec] ${text}`);
+        SuiteLocal.log(`[${msec}msec] ${text}`);
       });
     }
   }
 
   async after() {
     await rm(this.tempFolder, { recursive: true });
   }
+  static log(args: any) {
```

Comment:
Not sure I understand what this is doing?

Reply:
it's abstracting `console.log` so that it (`console.log`) doesn't appear in any of the source. That way I can grep for `console.log` and make sure I don't leave accidental messages anywhere.